src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
-body {
+body,
+p,
+h2,
+ul,
+span {
   margin: 0;
+  padding: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+@import "./block/card__container.scss";
+@import "./block/cards.scss";
+@import "./block/header.scss";
+@import "./block/nav.scss";
+@import "./block/stars.scss";
```

Comment:
U did a great job! But need some things to be fixed, try to check sizes and checklist one more time, i think u will find how to do everything to make npm tests work:)